packages/seller-client/src/components/BuyerDetails/RatingStars.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Box } from '@material-ui/core';
+import StarIcon from '@material-ui/icons/Star';
+import StarHalfIcon from '@material-ui/icons/StarHalf';
+import StarBorderIcon from '@material-ui/icons/StarBorder';
+
+const STARS = new Array(5).fill(1);
+
+type Props = {
+  rating: number;
+  size: number;
+};
+
+const RatingStars: React.FC<Props> = ({ rating, size }) => {
+  const style = { width: size, height: size, color: '#FFBA07' };
+  return (
+    <Box display="flex">
+      {STARS.map((_, index) => {
+        const starNumber = index + 1;
+        if (rating >= starNumber) {
+          return <StarIcon style={style} />;
+        } else if (starNumber - rating < 1) {
```

Comment:
Having the variables on the same side for each and using greater and greater than makes it easier for me to understand

```suggestion
        } else if (rating > starNumber - 1) {
```

Reply:
updated in 8d239d83817e469e5bb88b6324a34735b74cfd55